content/chef_compliance_phase.md

```diff
@@ -415,6 +419,29 @@ Setting the attribute `default['audit']['insecure']` to `true` will skip SSL cer
 default['audit']['insecure'] = true
 ```
 
+### interval
+
+**New in Chef Infra Client 17.8**
+
+You can control the frequency of Compliance Phase scans with the `default['audit']['interval']`, which means that you can set Compliance Phase scans to happen at different times than Chef Infra Client runs. This helps you control the impact of compliance scans on system performance in business environments that require compliance scans less frequently than Chef Infra Client Runs.
```

Comment:
less frequent interval is probably more clear. It's not a different time. It has to be at a time that the Infra Phase is running. It's just not every time the Infra Phase is running.